src/main/java/uk/gov/hmcts/juror/api/moj/service/jurormanagement/JurorAppearanceServiceImpl.java

```diff
@@ -196,6 +196,11 @@ public boolean hasAppearances(String jurorNumber) {
         return appearanceRepository.countByJurorNumber(jurorNumber) > 0;
     }
 
+    @Override
+    public boolean hasAttendances(String jurorNumber) {
+        return appearanceRepository.countByJurorNumberAndAppearanceStageNotNull(jurorNumber) > 0;
+    }
```

Comment:
Need unit tests

Reply:
it is part of the hasAppearances unit test although not tested in isolation